src/redux/session/actions.js

```diff
@@ -83,7 +83,10 @@ export const bootstrap = (relogin = true) => async (dispatch) => {
     return
   }
 
-  const localAccount = ls.getLocalAccount()
+  const state = getState()
+  const session = state.get(DUCK_SESSION)
+
+  const localAccount = session.account
```

Comment:
Need a help here
Do not sure if localAccount is a required.
Are there any cases when `localAccount` should not be equal to `account`?